src/App.tsx

```diff
@@ -1,61 +1,148 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+const getUser = (userId: number): User | null => {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const todosWithUsers: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+const getIdForTodo = (todos: Todo[]) => {
+  let biggest = 0;
```

Comment:
u better use more understanding naming, biggestId will be better

Reply:
agree, fix @avkovlit comment